lib/Onnxifi/CMakeLists.txt

```diff
@@ -3,8 +3,10 @@ include_directories(${GLOW_THIRDPARTY_DIR}/onnx)
 
 add_library(onnxifi-glow
             SHARED
+              base.cpp
```

Comment:
Nit: capital B?  (Unless it needs to be lowercase for special onnxy reasons)

Reply:
nope, no special reasons here.